hack/remove-calico-policy/README.md

```diff
@@ -1,4 +1,41 @@
-# Disabling and Removing Calico Policy
+# Disabling Calico Policy
+
+This guide explains two methods for disabling Calico policy on a running cluster. These steps are intended as a
+last resort should it appear that Calico policy is not functioning properly on a production system.
+
+- [Overriding Calico Policy] describes how to add high-priority Calico System Network Policy which
+  overrides the configured Network Policy.
+- [Disabling and Removing Calico Policy] describes how to fully disable Calico policy (removing all 
+  Calico-programmed policy from each node.
+
+## Overriding Calico Policy
```

Comment:
Gr, let's make this doc the big switch.

- README.md
  - override-policy.md
  - remove-policy.md

README.md can contain the intro / context bits.